### PR TITLE
Updated SageMaker examples to take a region arg; added doc comments.

### PR DESCRIPTION
**Pull request by Doug-AWS**
*Issue #, if available:*
SageMaker code examples had no doc comments.

*Description of changes:*
Added them and an optional command-line region arg (-d DEFAULT-REGION).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/sagemaker/src/bin/list-training-jobs.rs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        let duration = training_end_time - creation_time;
+
+        let deets = format!(
+            "{}\t{}\t{}\t{:#?}",
+            name,
+            creation_time.format("%Y-%m-%d@%H:%M:%S"),
+            duration.num_seconds(),
+            status
+        );
+        println!("{}", deets);
```
Comment (jdisanti) [inline aws/sdk/examples/sagemaker/src/bin/list-training-jobs.rs:71]: You can combine these two lines:
```
println!(
    "{}\t{}\t{}\t{:#?}",
    name,
    ...
);
```

File: aws/sdk/examples/sagemaker/src/bin/sagemaker-helloworld.rs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    for n in notebooks.notebook_instances.unwrap_or_default() {
+        let n_instance_type = n.instance_type.unwrap();
+        let n_status = n.notebook_instance_status.unwrap();
+        let n_name = n.notebook_instance_name.as_deref().unwrap_or_default();
+
+        let details = format!(
+            "Notebook Name : {}, Notebook Status : {:#?}, Notebook Instance Type : {:#?}",
+            n_name, n_status, n_instance_type
+        );
+        println!("{}", details);
```
Comment (jdisanti) [inline aws/sdk/examples/sagemaker/src/bin/sagemaker-helloworld.rs:63]: Likewise here

**Review by jdisanti — CHANGES_REQUESTED** [review]

File: aws/sdk/examples/sagemaker/src/bin/list-training-jobs.rs (hunk 1 above)
Comment (Doug-AWS) [reply aws/sdk/examples/sagemaker/src/bin/list-training-jobs.rs:71]: Updated, thanks.

File: aws/sdk/examples/sagemaker/src/bin/sagemaker-helloworld.rs (hunk 2 above)
Comment (Doug-AWS) [reply aws/sdk/examples/sagemaker/src/bin/sagemaker-helloworld.rs:63]: Done.

**Review by jdisanti — APPROVED** [review]

**Pull request merged**